index.bs

```diff
@@ -1811,6 +1825,7 @@ navigation status</dfn> struct, which has the following items:
 
 <pre class="cddl remote-cddl">
 BrowsingContextCommand = (
+; keep-sorted start
```

Comment:
```suggestion
  ; keep-sorted start
```

Interesting idea. Also could we actually have a default set so that we do not have to repeat it in all the cases?

Reply:
If using the keep-sorted package, unfortunately this is the only way (repeating it in all cases).

A more robust solution would be to have a bikeshed formatter that automatically does so (a la python-isort), but I'm not aware of any.